samples/zigbee/light_bulb/src/zcl_scenes.c

```
@@ -0,0 +1,939 @@
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+			buf,
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_ON_OFF,
+			attr_desc,
+			&entry->on_off.on_off,
+			result
+		);
+	}
+	if (entry->level_control.has_current_level) {
+		LOG_INF("Recall level control state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_LEVEL_CONTROL,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_LEVEL_CONTROL_CURRENT_LEVEL_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+			buf,
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_LEVEL_CONTROL,
+			attr_desc,
+			&entry->level_control.current_level,
+			result
+		);
+	}
+	if (entry->window_covering.has_current_position_lift_percentage) {
+		LOG_INF("Recall window covering lift state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_WINDOW_COVERING_CURRENT_POSITION_LIFT_PERCENTAGE_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+				buf,
+				ZCL_SCENES_ENDPOINT,
+				ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+				attr_desc,
+				&entry->window_covering.current_position_lift_percentage,
+				result
+		);
+	}
+	if (entry->window_covering.has_current_position_tilt_percentage) {
+		LOG_INF("Recall window covering tilt state");
+
+		attr_desc = zb_zcl_get_attr_desc_a(
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+			ZB_ZCL_CLUSTER_SERVER_ROLE,
+			ZB_ZCL_ATTR_WINDOW_COVERING_CURRENT_POSITION_TILT_PERCENTAGE_ID);
+
+		ZB_ZCL_INVOKE_USER_APP_SET_ATTR_WITH_RESULT(
+			buf,
+			ZCL_SCENES_ENDPOINT,
+			ZB_ZCL_CLUSTER_ID_WINDOW_COVERING,
+			attr_desc,
+			&entry->window_covering.current_position_tilt_percentage,
+			result
+		);
+	}
+
+	zb_buf_free(buf);
+}
+
+static void send_view_scene_resp(zb_bufid_t bufid, zb_uint16_t idx)
+{
+	zb_uint8_t *payload_ptr;
+	zb_uint8_t view_scene_status = ZB_ZCL_STATUS_NOT_FOUND;
+
+	LOG_INF(">> %s bufid %hd idx %d", __func__, bufid, idx);
```

Wouldn't it be better to have these entry/exit logs at debug level? INF is the default log level and it looks that these logs could obscure the output a bit.

---

Entry-to-function and exit logs changed from INF to DBG